<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SchemeEditorMW</class>
 <widget class="QMainWindow" name="SchemeEditorMW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Energo - Редактор схем</string>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>541</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="mainLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,0">
       <property name="margin">
        <number>6</number>
       </property>
       <item>
        <widget class="QGraphicsView" name="graphicsView">
         <property name="backgroundBrush">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>200</red>
            <green>200</green>
            <blue>200</blue>
           </color>
          </brush>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layoutSettings">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTableWidget" name="twSettings">
           <property name="showDropIndicator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>true</bool>
           </attribute>
           <attribute name="horizontalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>20</number>
           </attribute>
           <column>
            <property name="text">
             <string>Наименование</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Значение</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblHelp">
           <property name="text">
            <string>Нет помощи.</string>
           </property>
           <property name="textFormat">
            <enum>Qt::LogText</enum>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>аспс</string>
     </property>
     <addaction name="action_6"/>
    </widget>
    <addaction name="menu_3"/>
    <addaction name="actionDfdf"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Элементы</string>
    </property>
    <addaction name="actMeter"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpen"/>
   <addaction name="actSave"/>
   <addaction name="separator"/>
   <addaction name="actEditState"/>
   <addaction name="separator"/>
   <addaction name="actTextObject"/>
   <addaction name="actDbLink"/>
   <addaction name="actOilBreaker"/>
   <addaction name="actBreaker"/>
   <addaction name="separator"/>
   <addaction name="actEllipse"/>
   <addaction name="actRect"/>
   <addaction name="actTriangle"/>
   <addaction name="actLine"/>
   <addaction name="separator"/>
   <addaction name="actConnectionLine"/>
   <addaction name="separator"/>
  </widget>
  <action name="actDbLink">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/sceditor/toolbox/objects/database-link.png</normaloff>:/sceditor/toolbox/objects/database-link.png</iconset>
   </property>
   <property name="text">
    <string>Источник данных</string>
   </property>
  </action>
  <action name="actTextObject">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/sceditor/toolbox/objects/case_smallcaps.png</normaloff>:/sceditor/toolbox/objects/case_smallcaps.png</iconset>
   </property>
   <property name="text">
    <string>Тестовый элемент</string>
   </property>
  </action>
  <action name="actEditState">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/sceditor/toolbox/objects/applications-development-5.png</normaloff>:/sceditor/toolbox/objects/applications-development-5.png</iconset>
   </property>
   <property name="text">
    <string>Режим дизайнера</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actPreviewState">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/sceditor/toolbox/objects/document-preview.png</normaloff>:/sceditor/toolbox/objects/document-preview.png</iconset>
   </property>
   <property name="text">
    <string>Режим просмотра</string>
   </property>
  </action>
  <action name="actSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/sceditor/toolbox/objects/document-save-5.png</normaloff>:/sceditor/toolbox/objects/document-save-5.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/sceditor/toolbox/objects/document-open-2.png</normaloff>:/sceditor/toolbox/objects/document-open-2.png</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actEllipse">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/sceditor/toolbox/objects/draw-ellipse-2.png</normaloff>:/sceditor/toolbox/objects/draw-ellipse-2.png</iconset>
   </property>
   <property name="text">
    <string>Эллипс</string>
   </property>
  </action>
  <action name="actConnectionLine">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/sceditor/toolbox/objects/draw-bezier-curves.png</normaloff>:/sceditor/toolbox/objects/draw-bezier-curves.png</iconset>
   </property>
   <property name="text">
    <string>Соединительная линия</string>
   </property>
  </action>
  <action name="actRect">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/sceditor/toolbox/objects/draw-rectangle-2.png</normaloff>:/sceditor/toolbox/objects/draw-rectangle-2.png</iconset>
   </property>
   <property name="text">
    <string>Квадрат</string>
   </property>
  </action>
  <action name="actTriangle">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/sceditor/toolbox/objects/draw-triangle3.png</normaloff>:/sceditor/toolbox/objects/draw-triangle3.png</iconset>
   </property>
   <property name="text">
    <string>Треугольник</string>
   </property>
  </action>
  <action name="actLine">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/sceditor/toolbox/objects/draw-line-3.png</normaloff>:/sceditor/toolbox/objects/draw-line-3.png</iconset>
   </property>
   <property name="text">
    <string>Прямая</string>
   </property>
  </action>
  <action name="actOilBreaker">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/sceditor/objects/objects/oil-circuit-breaker.PNG</normaloff>:/sceditor/objects/objects/oil-circuit-breaker.PNG</iconset>
   </property>
   <property name="text">
    <string>Маслянный выключатель</string>
   </property>
  </action>
  <action name="actBreaker">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/sceditor/objects/objects/breaker.PNG</normaloff>:/sceditor/objects/objects/breaker.PNG</iconset>
   </property>
   <property name="text">
    <string>Выключатель</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Трансформатор тока</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Трансформатор напряжения</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Тест</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>ап</string>
   </property>
  </action>
  <action name="actMeter">
   <property name="text">
    <string>Стрелочный прибор</string>
   </property>
  </action>
  <action name="actionDfdf">
   <property name="text">
    <string>dfdf</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
